Description: Drop git requirement
 Only to allow local build without git.
Author: Sophie Brun <sophie@freexian.com>
Last-Update: 2018-07-06
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/build_scripts/main.py
+++ b/build_scripts/main.py
@@ -299,9 +299,9 @@
 
 
 def prepare_build():
-    if (os.path.isdir(".git") and not OPTION_IGNOREGIT and
-            not OPTION_ONLYPACKAGE and not OPTION_REUSE_BUILD):
-        prepare_sub_modules()
+    #if (os.path.isdir(".git") and not OPTION_IGNOREGIT and
+    #        not OPTION_ONLYPACKAGE and not OPTION_REUSE_BUILD):
+    #    prepare_sub_modules()
     # Clean up temp build folder.
     for n in ["build"]:
         d = os.path.join(setup_script_dir, n)
